fix import for  random_string & random_password resource (#256)
hashicorp/terraform-provider-random: CHANGELOG.md

> ## 3.3.2 (June 08, 2022)

Reviewer: ```suggestion
## 3.3.2 (unreleased)
```

BUG FIXES:

> * resource/random_password: When importing set defaults for `lower`, `upper`, `special`, `number`, `numeric`, `min_lower`, `min_upper`, `min_special` and `min_numeric` ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).

Reviewer: ```suggestion
* resource/random_password: When importing set defaults for all attributes that have a default defined ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).
```

> * resource/random_password: When importing write all defaults into state file([256](https://github.com/hashicorp/terraform-provider-random/pull/256))
> * resource/random_string: When importing write all defaults into state file([256](https://github.com/hashicorp/terraform-provider-random/pull/256))

Reviewer: ```suggestion
* resource/random_password: When importing set defaults for `lower`, `upper`, `special`, `number`, `numeric`, `min_lower`, `min_upper`, `min_special` and `min_numeric` ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).
* resource/random_string: When importing set defaults for `lower`, `upper`, `special`, `number`, `numeric`, `min_lower`, `min_upper`, `min_special` and `min_numeric` ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).
```

> * resource/random_string: When importing set defaults for `lower`, `upper`, `special`, `number`, `numeric`, `min_lower`, `min_upper`, `min_special` and `min_numeric` ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).

Reviewer: ```suggestion
* resource/random_string: When importing set defaults for all attributes that have a default defined ([256](https://github.com/hashicorp/terraform-provider-random/pull/256)).
```
